sphinx_scality/static/css/current.css_t

```diff
@@ -36,7 +36,7 @@ p {
 
 div.baselogo > a {
     display: flex;
-    font-family: Roboto, sans-serif;
+    font-family: Arial;
```

Comment:
Keep the fallback font, in case Arial is not available (not usual, but could happen in theory).

```suggestion
    font-family: Arial, sans-serif;
```